dags/common/cleanup.py

```diff
@@ -1,11 +1,17 @@
 def clean_whitespace_characters(input):
     return " ".join(input.split())
 
+
 def convert_html_subsripts_to_latex(input):
     from re import sub
 
     input = sub("<sub>(.*?)</sub>", r"$_{\1}$", input)
     input = sub("<inf>(.*?)</inf>", r"$_{\1}$", input)
-    input = sub("<sup>(.*?)</sup>", r"$^{\1}$", input)   
+    input = sub("<sup>(.*?)</sup>", r"$^{\1}$", input)
 
     return input
+
+
+def clean_collaboration(input):
+    """Remove the prefixes for collaborations"""
```

Comment:
@ErnestaP This was stuck as a pending review, sorry for the late reply

Reply:
I see your point. I agree that code has to be self-explanatory and this is a docstring, which just explains what functions do. For me, because I don't know what collaboration string looks like, gives me a bit of sense of why we have to remove these two words.
I would add docstring on almost all functions. That is what I was taught. Of course, this docstring which is in this PR is not the best example.